src/main/scripts/org/reaktivity/specification/nukleus/kafka/streams/merged/unmerged.fetch.server.sent.abort.with.message/server.rpt

```diff
@@ -0,0 +1,147 @@
+                                  .config("max.message.bytes", 1000012)
+                                  .config("segment.bytes", 1073741824)
+                                  .config("segment.index.bytes", 10485760)
+                                  .config("segment.ms", 604800000)
+                                  .config("retention.bytes", -1)
+                                  .config("retention.ms", 604800000)
+                                  .config("delete.retention.ms", 86400000)
+                                  .config("min.compaction.lag.ms", 0)
+                                  .config("max.compaction.lag.ms", 9223372036854775807)
+                                  .config("min.cleanable.dirty.ratio", 0.5)
+                                  .build()
+                              .build()}
+write flush
+
+accepted
+
+read nukleus:begin.ext ${kafka:beginEx()
+                              .typeId(nukleus:id("kafka"))
+                              .meta()
+                                  .topic("test")
+                                  .build()
+                              .build()}
+
+connected
+
+write nukleus:begin.ext ${kafka:beginEx()
+                               .typeId(nukleus:id("kafka"))
+                               .meta()
+                                   .topic("test")
+                                   .build()
+                               .build()}
+write flush
+
+write nukleus:data.ext ${kafka:dataEx()
+                              .typeId(nukleus:id("kafka"))
+                              .meta()
+                                  .partition(0, 1)
+                                  .build()
+                              .build()}
+write flush
+
+accepted
+
+read nukleus:begin.ext ${kafka:beginEx()
+                              .typeId(nukleus:id("kafka"))
+                              .fetch()
+                                  .topic("test")
+                                  .partition(0, -2)
+                                  .build()
+                              .build()}
+
+connected
+
+write nukleus:begin.ext ${kafka:beginEx()
+                               .typeId(nukleus:id("kafka"))
+                               .fetch()
+                                   .topic("test")
+                                   .partition(0, 1)
+                                   .build()
+                               .build()}
+write flush
+
+write nukleus:data.ext ${kafka:dataEx()
+                              .typeId(nukleus:id("kafka"))
+                              .fetch()
+                                  .timestamp(newTimestamp)
+                                  .partition(0, 1)
+                                  .key("key1")
+                                  .header("header2", "value2")
+                                  .build()
+                              .build()}
+write "Hello, world #A1"
+write flush
+
+write await ABORT_FETCH
```

Comment:
For readability, this `write await` should be moved down closer to `write abort`.
Also, suggest renaming this barrier to `ABORT_UNMERGED_FETCH_REPLY`.